deploy/docker-compose/docker-compose-fixed-instrumented-attack.yml

```diff
@@ -65,6 +65,27 @@ services:
     labels:
       com.datadoghq.ad.logs: '[{"source": "ruby", "service": "store-frontend"}]'
       my.custom.label.team: "frontend"
+  nginx:
+    environment:
+      - DD_AGENT_HOST=agent
+      - DD_LOGS_INJECTION=true
+      - DD_TRACE_ANALYTICS_ENABLED=true
+      - DD_SERVICE=store-frontend
+      - DB_USERNAME
+      - DB_PASSWORD
+      - DD_VERSION=1.1
+      - DD_CLIENT_TOKEN
+      - DD_APPLICATION_ID
+      - DD_ENV=development
+      - DD_SITE=datadoghq.com
+    image: "ddtraining/nginx:2.1.0"
```

Comment:
this isn't a valid tag because nginx hasn't been pushed to the repo yet as part of a tagged release. After this PR is merged in + release is published (2.1.3) - I'll create a separate PR to update the pinned images to use the new tag

Reply:
understood.  Nothing is using the attack-box compose yet.  So we should be able to move forward.

Reply:
Also these environment variables are not used by nginx. The compose nginx config should match what exists in `docker-compose-local-baseline`